AdyenDropIn/Utilities/ComponentManager.swift

```diff
@@ -80,7 +80,6 @@ internal final class ComponentManager {
             return nil
         }
         
-        cardComponent?.showsLargeTitle = false
```

Comment:
default value is `true`
I believe you have fixed and tested it everywhere?

Reply:
in those components, they check if `showsLargeTitle` is true AND `_isDropIn` is false, so this should cover this scenario.

Reply:
👍